### ISSUE TITLE
Providing good documentations with examples for beginners

### ISSUE THREAD

**Issue by zuhhaga** [openSUSE/libzypp#475]
Is there good documentations for beginners? Thanks

**Comment by mlandres** [comment]
Unfortunately not. There's the [doxygen generated](https://doc.opensuse.org/projects/libzypp/HEAD/) docu (also available in the libzypp-devel-doc package). It probably lacks an entrypoint for beginners.

**Comment by zuhhaga** [comment]
Yes, docu lacks an entrypoint for beginners, that's very sad. For example, if someone wish to contribute to libzypp or create project using libzypp, he/she, probably, will examine projects that are already written using libzypp, or, in common cases, use alternative engine, that have some entrypoints for beginners on their documentations.
